### PR TITLE
fix platform.json on Wolverine for thermal sensors

### PR DESCRIPTION
**Pull request by patrickmacarthur**
#### Why I did it

Manual rebase of PR #13524 to 202211 branch.

#### How I did it

See PR #13524

### CODE REVIEW

File: device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
+    "chassis": {
+        "name": "7800R3-36DM2-LC",
```
Comment (wenyiz2021) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json:3]: should be "7800R3A-36DM2-LC"?

**Review by wenyiz2021 — CHANGES_REQUESTED** [review]

File: device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json (hunk 1 above)
Comment (patrickmacarthur) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json]: fixed

File: device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
+{
+    "chassis": {
+        "name": "7800R3A-36DM2-LC",
+        "components": [],
```
Comment (wenyiz2021) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json:4]: "components": [
            {
                "name": "Aboot()"
            },
            {
                "name": "Scd(addr=0000:00:18.7)"
            },
            {
                "name": "Scd(addr=0000:08:00.0)"
            }
        ],

**Comment by wenyiz2021** [comment]
please double check if the file is exactly same as: https://github.com/sonic-net/sonic-buildimage/pull/13524/files

File: device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json (hunk 2 above)
Comment (patrickmacarthur) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json]: apologies; I guess I had an outdated version of the patch. I just pushed a file that should be identical to master.

**Review by wenyiz2021 — APPROVED** [review]

**Review by Staphylo — APPROVED** [review]

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**